glossary.md

```diff
@@ -157,6 +157,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | negative sample/example           | mẫu âm                                                         |                                             |
 | neural network                    | mạng nơ-ron                                                    | http://bit.ly/2BvfPYA http://bit.ly/2MAkizG |
 | norm                              | chuẩn                                                          |                                             |
+| null hypothesis                   | giả thuyết rỗng                                                          |                                             |
```

Comment:
Các từ liên quan tới xác suất thống kê nên dựa theo cuốn [Giáo trình xác suất thống kê -- Tống Đình Quỳ](https://sachvui.com/ebook/giao-trinh-xac-suat-thong-ke-tong-dinh-quy.3129.html). Đây là giáo trình của ĐHBKHN đã được dùng trong nhiều năm.

Trang 158 là bắt đầu chương "Kiểm định giả thuyết" (hypothesis testing), trong đó "null hypothesis", thường được ký hiệu là H_0, được dịch là "giả thuyết gốc".

Reply:
Từ null này còn dịch là rỗng nữa. Bên compressive sensing có dùng Null Space, mình chắc chắn nó không có nghĩa là không gian gốc. 

Phải thảo luận thêm từ này

Reply:
À, tất nhiên `null` đứng một mình nghĩa là `rỗng` hay `không`. Nhưng trong ngữ cảnh này thì `gốc` hoàn toàn có nghĩa. Đặc biệt là khi nó đã được sử dụng trong các giáo trình.

Trong thống kê, `null hypothesis` là giả thuyết mà ta cần kiểm định nó có đúng theo cách hiểu thống kê không. Sau khi tính toán ra một phải con số thì ta có thể kết luận chấp nhận hay phủ nhận giả thuyết này. Đi kèm với cụm này là `alternative hypothesis`, được dịch là `giả thuyết đối` hoặc `đối thuyết`. Một số tài liệu dịch là `giả thuyết không` và `giả thuyết nghịch`. Mình có thể mở ngoặc cả hai cách dịch nhưng sử dụng một cho thống nhất. Từ "null" có thể dịch theo ngữ cảnh, nhưng cả cụm "null hypothesis" thì chỉ nên có một cách dịch mà thôi.

Mình vừa search nhanh thì thấy "giả thuyết không"  được sử dụng nhiều hơn.

Cụm "null space" không chỉ trong compressive sensing mà nó là cơ bản trong đại số tuyến tính rồi. Ngày trước học mình nhớ `null space` được dịch là `không gian null`.

Reply:
Nếu đã có giáo trình thì phổ biến thì mình cứ theo giáo trình, vì đây cũng không phải là từ khoá ML mới, mình chỉ cần thống nhất thôi.
```suggestion
| null hypothesis                   | giả thuyết gốc                                                          |                                             |
```